### DISCUSSION TITLE
Error while running "symfony console make:user"

### DISCUSSION THREAD

**Discussion by Caikozolyv** [symfony/symfony#42400] [Q&A] [answered]
I got the error 
![image](https://user-images.githubusercontent.com/25210744/128490204-4910cbcb-b568-4025-9f58-2377fda5c34d.png)
while running symfony console make:user

I tried updating php from version 7.4 to 8.0 but I still get the same error.

**Comment by axzx** [comment]
See what you have in the Repository.tpl.php file.
Why do you have the function str_contains there?

**Reply by Caikozolyv** [reply]
This is a file from : [symfony/maker-bundle ](https://github.com/symfony/maker-bundle/blob/main/src/Resources/skeleton/doctrine/Repository.tpl.php)
I didn't modify it

**Reply by axzx** [reply]
https://www.php.net/manual/en/function.str-contains
(PHP 8)

show 
`php -v`

**Reply by javiereguiluz** [reply]
That function was added in PHP 8 ... you can use it in any previous PHP version too if you install this package: https://github.com/symfony/polyfill-php80

I don't know the internals of Maker bundle well ... but it's strange that it uses a PHP 8 function without requiring the PHP 8 polyfill.

Maybe it's better to report this as a possible bug in their repository: https://github.com/symfony/maker-bundle/issues

**Reply by Caikozolyv** [reply]
Installing polyfill-php80 and changing php version to 7.4 did the job. 
Can you make your response as comment that I mark it as answer ?
I'll report it later too

**Comment by javiereguiluz** [comment, marked as answer]
That function was added in PHP 8 ... you can use it in any previous PHP version too if you install this package: https://github.com/symfony/polyfill-php80

I don't know the internals of Maker bundle well ... but it's strange that it uses a PHP 8 function without requiring the PHP 8 polyfill.

Maybe it's better to report this as a possible bug in their repository: https://github.com/symfony/maker-bundle/issues
